.github/_typos.toml

```diff
@@ -23,7 +23,8 @@ extend-exclude = [
     "PopulationByCountry.csv",
     "PopulationByAdmin1.csv",
     "WomensSuffrage.txt",
-    "SK-dotnet.sln.DotSettings"
+    "SK-dotnet.sln.DotSettings",
+    "dotnet/samples/Concepts/Resources/Plugins/CopilotAgentPlugins/README.md"
```

Comment:
I wonder what was detected as a typo in the README and whether the whole file should be excluded?

Reply:
Follow up https://github.com/microsoft/semantic-kernel/issues/9898